experimental/dds/tree2/src/test/feature-libraries/editable-tree-2/editableTreeTypes.spec.ts

```diff
@@ -0,0 +1,206 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert } from "assert";
+import { unreachableCase } from "@fluidframework/core-utils";
+import {
+	jsonArray,
+	jsonBoolean,
+	jsonNull,
+	jsonNumber,
+	jsonObject,
+	jsonRoot,
+	jsonSchema,
+	jsonString,
+} from "../../../domains";
+
+import {
+	Sequence,
+	TypedNode,
+	UntypedField,
+	ValueField,
+	// eslint-disable-next-line import/no-internal-modules
```

Comment:
Is there a reason we have to drill down into the inner directory here? Can we document it? 😋

Reply:
we do this for the file that is being tested in most of our test files. I tried documenting it every time I did it, but that pattern didn't stick.

The case where we do it below is more  questionable as that is not the file being tested, and should probably be fixed.

Reply:
This is one of the reasons why I prefer allowing tests at all levels of the package hierarchy, rather than keeping them strictly at the root. Obviously not for this PR, but that is a pattern I would love to see change across the repo.

Reply:
There are different downsides to that approach, though I generally agree its a bit better. Not as nice as being able to test in the same file like rust does, but we can't have nice things.